### PR TITLE
Globals for Fiber

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

File: panel/src/fiber/app.js [hunk 1]
```diff
@@ -14,24 +14,32 @@ export default {
     Fiber.init({
       page: window.fiber,
       component: (name) => Vue.component(name),
-      props: (props) => {
+      props: (data) => {
+
+        /** Set globals */
+        ["$config", "$system", "$translation", "$urls"].forEach(key => {
```
Comment (distantnative) [inline panel/src/fiber/app.js]: Why are we setting these differently than the others?
I don't think you'll need the else-statement here. Our `protocol.js` keeps the globals from the initial request around and adds them after every partial request anyways. So this method sees always a full `data` object.

Comment (bastianallgeier) [reply panel/src/fiber/app.js]: I didn't get it running without this if clause.

File: src/Panel/Panel.php [hunk 2]
```diff
@@ -228,10 +228,11 @@ public static function customJs(App $kirby)
     public static function data(App $kirby, string $component, array $data = []): array
     {
         // shared default data
+        $multilang = $kirby->option('languages', false) !== false;
```
Comment (afbora) [inline src/Panel/Panel.php:231]: If I haven't missed anything, why don't we use the `$kirby->multilang()` method? 🤔

Comment (distantnative) [reply src/Panel/Panel.php:231]: I think the definition is a bit different: `$kirby->multilang()` checks if there actually has been at least one language defined. While we here are checking if the config allows defining languages - no matter if the first one has been defined or not.

Comment (afbora) [reply src/Panel/Panel.php:231]: Ah right 👍

**Pull request merged**